app/ports/migrations/0013_buildhistory_add_index.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 2.2.10 on 2020-04-13 16:08
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ports', '0012_add_10_15_worker'),
+    ]
+
+    operations = [
+        migrations.AddIndex(
+            model_name='buildhistory',
+            index=models.Index(fields=['port_name', 'builder_name', '-time_start'], name='ports_build_port_na_5f3089_idx'),
```

Comment:
Just to clarify: so this index already exists in the model file, it was just forgotten to be added as a migration?

Reply:
Yes, I checked the file history which revealed that no migration was added after adding this index. Also when I ran the command locally, this migration was generated.